build/native/cargo_driver.rs

```diff
@@ -654,3 +656,29 @@ pub fn to_cmake_path_list(iter: impl IntoIterator<Item = impl AsRef<OsStr>>) ->
     }
     Ok(accu)
 }
+
+fn copy_binaries_to_target_folder() {
```

Comment:
I simply don't want the whole build to fail simply because it failed to copy two files. For most users it will still be completely fine if those files are not there, and to me it just doesn't make sense that the entire build should fail because of that.

Reply:
... which also means once/if it stops working, it will just go unnoticed, which is not fine either.